calico/openstack/mech_calico.py

```diff
@@ -150,11 +154,16 @@ def __init__(self):
         # Initialize fields for the database object and transport.  We will
         # initialize these properly when we first need them.
         self.db = None
```

Comment:
Why isn't db_context a private attribute? It's not accessed anywhere outside the class.